packages/framework/tests/Unit/Model/Newsletter/NewsletterFacadeTest.php

```diff
@@ -14,17 +14,17 @@
 class NewsletterFacadeTest extends TestCase
 {
     /**
-     * @var EntityManager|MockObject
+     * @var \Doctrine\ORM\EntityManager|MockObject
      */
     private $em;
 
     /**
-     * @var NewsletterRepository|MockObject
+     * @var \Shopsys\FrameworkBundle\Model\Newsletter\NewsletterRepository|MockObject
```

Comment:
`MockObject` should be FQNized as well probably, responsibility of Slevomat sniff